{"name": "test_login_fail[fail_data3]", "status": "passed", "description": "\n        用例描述：测试登录失败用例\n        ", "attachments": [{"name": "log", "source": "8bf3bb80-2877-47f8-b173-ed9eabf9907b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "b7890d87-32df-481d-8463-8c748c921178-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "fail_data", "value": "{'sUserName': '16533907260', 'sPassword': ''}"}], "start": 1584083434463, "stop": 1584083435143, "uuid": "d9c3c273-554d-4661-ba88-8b5e7ddc5892", "historyId": "db57e921a10ef9659acaa552a08558fa", "testCaseId": "8000cb86548fa2cebd0d986e95a5b795", "fullName": "test_case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "登录失败用例"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "10400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}